Polyhedron/demo/Polyhedron/Plugins/Point_set/Surface_reconstruction_plugin.cpp

```diff
@@ -621,7 +625,7 @@ void Polyhedron_demo_surface_reconstruction_plugin::automatic_reconstruction
 	      time.restart();
 
 	      Scene_polyhedron_item* reco_item = new Scene_polyhedron_item(Polyhedron());
-	      SurfaceReconstruction::advancing_front (*points, reco_item, (std::max)(noise_size, aniso_size));
+	      SurfaceReconstruction::advancing_front (*points, reco_item, 10. * (std::max)(noise_size, aniso_size));
 	      
```

Comment:
Yes. It used to be in the general function _SurfaceReconstruction::advancing_front_ (see line 378) but that was a bug (it should be multiplied by 10 _only_ when parameters are estimated like here - NOT when it is user-defined, which would be misleading).